rpi3/firmware/optee

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
I'll remove it. Sounds like we're on the same page here. Phil can add it later on if needed.